src/main/java/edu/umich/tl/Utils.java

```diff
@@ -68,5 +70,13 @@ public static InputStream getInputStreamFromURL(String fileURL) {
 	public static boolean isEmpty(String value) {
 		return (value == null) || (value.trim().equals(""));
 	}
+	// iso8601String= "2015-01-08T23:00:00Z"
+	public static DateTime changeToDate(String iso8601String) {
```

Comment:
Very good to use UTC.  That can avoid many problems.  Are ALL dates processed in UTC?